lib/eChronicler_web/controllers/hello_controller.ex

```diff
@@ -0,0 +1,12 @@
+
+defmodule EChroniclerWeb.HelloController do
+  use EChroniclerWeb, :controller
+
+  def index(conn, _params) do
+    render(conn, "index.html")
+  end
+
+  def show(conn, %{"messenger" => messenger}) do
+    render(conn, "show.html", messenger: messenger)
+  end
```

Comment:
See comment below.  Is this vestigial/unused?

Later: Ah, I see, this is part of the HelloController, which was perhaps an area for experimentation.  Still, the comment below still applies -- you should remove things that will be dead code shortly, and try to limit the scope of the PR to the list-all journal entries functionality.  See comment below for other suggestions.

Reply:
Done!